libraries/botbuilder-dialogs/src/prompts/choicePrompt.ts

```diff
@@ -106,6 +106,9 @@ export class ChoicePrompt extends Prompt<FoundChoice> {
         const result: PromptRecognizerResult<FoundChoice> = { succeeded: false };
         const activity: Activity = context.activity;
         const utterance: string = activity.text;
+        if (utterance == null) {
```

Comment:
Consider 

`if (!utterance) `

if you want to handle the undefined and zero-length cases as well.